### ISSUE TITLE
There should be a way to announce things in the semantics layer

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#1720]
e.g. consider the gallery app progress indicators page. The accessibility layer should be told when the text changes so that it can read it out.

**Comment by goderbauer** [comment]
@amirh Does your `announce` implementation address this?

**Comment by amirh** [comment]
I'm not sure whether @Hixie referred to integration with platform mechanism that announce changes in the accessibility tree.

https://github.com/flutter/flutter/pull/12594 adds a way to do any ad-hoc announcements
